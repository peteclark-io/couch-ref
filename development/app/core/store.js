'use strict';

import { createStore } from 'redux';
import reducer from '../ducks/reducers';

const store = createStore(reducer);

export default store;
