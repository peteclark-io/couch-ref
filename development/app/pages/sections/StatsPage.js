'use strict'

import React from 'react';

import QuestionResults from '../../components/Stats/QuestionResults';

const StatsPage = React.createClass({

   render: function() {
      return (
         <QuestionResults id={this.props.params.questionId} />
      );
   }
});

export default StatsPage;
