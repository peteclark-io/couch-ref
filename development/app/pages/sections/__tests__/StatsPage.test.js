'use strict';

import React from 'react';
import StatsPage from '../StatsPage';
import {shallow} from 'enzyme';

import {Provider} from 'react-redux';
import store from '../../../core/store';

test('StatsPage section', () => {
   var params = {questionId: 1201}
   const rendered = shallow(
      <Provider store={store}>
         <StatsPage params={params} />
      </Provider>
   );

   let tree = rendered.debug();
   expect(tree).toMatchSnapshot();
});
