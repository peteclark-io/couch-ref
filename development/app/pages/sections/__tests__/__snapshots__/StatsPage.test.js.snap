exports[`test StatsPage section 1`] = `"<StatsPage params={{...}} />"`;
